sources/packages/backend/apps/queue-consumers/src/services/student-application-notifications/student-pd-ppd-reminder-notification.ts

```diff
@@ -0,0 +1,54 @@
+import { Injectable } from "@nestjs/common";
+import {
+  NotificationActionsService,
+  StudentPDPPDNotification,
+} from "@sims/services";
+import { ApplicationService } from "../../services";
+import { ProcessSummary } from "@sims/utilities/logger";
+
+/**
+ * Creates a student email notification - PD/PPD Student reminder email 8 weeks before end date.
+ */
+@Injectable()
+export class StudentPdPpdReminderNotification {
+  constructor(
+    private readonly applicationService: ApplicationService,
+    private readonly notificationActionsService: NotificationActionsService,
+  ) {}
+
+  /**
+   * Creates a student notification for the given PD/PPD Student reminder.
+   * @param processSummary process summary for logging.
+   */
+  async createNotification(processSummary: ProcessSummary): Promise<void> {
+    const eligibleApplications =
+      await this.applicationService.getApplicationWithPDPPStatusMismatch();
```

Comment:
Minor, not part of PR but the method is missing a "D". The method name should be 
`getApplicationWithPDPPDStatusMismatch`.